samples/sampler/stories/default/ProgressBar.js

```diff
@@ -21,6 +22,7 @@ export const _ProgressBar = (args) => (
 		progress={args['progress']}
 		progressAnchor={args['progressAnchor']}
 		showAnchor={args['showAnchor']}
+		style={args['orientation'] === 'vertical' || args['orientation'] === 'radial' ? {marginLeft: ri.scaleToRem(126), marginRight: ri.scaleToRem(120)} : null}
```

Comment:
It seems too spacy. Also, Why don't we use `className` instead of `style`?
How about 
```suggestion
		style={args['orientation'] === 'vertical' || args['orientation'] === 'radial' ? css.margin : null}
```
And add ProgressBar.module.less with

```
// ProgressBar.module.less
//

.margin {
	margin: 0 144px;
}

```